components/contact/ContactProvinceRow.js

```diff
@@ -16,7 +16,9 @@ const ContactProvinceRow = ({ label, items, id }) => {
         >
           {items.map((x, i) => (
             <div className="col-span-2 md:col-span-1 py-3" key={i}>
-              <p className="font-bold font-display">{x && x.content}</p>
+              <div className="prose prose-p:text-xl prose-p:font-display font-bold">
+                <Markdown>{`${ap(x.content, ' ')}`}</Markdown>
+              </div>
```

Comment:
Just a small change. We can remove the surrounding `<div>` and simply have this:

`<Markdown className="prose prose-p:text-xl prose-p:font-display font-bold">{`${ap(x.content, ' ')}`}</Markdown>`

Reply:
I can make it a span if that's preferable but it's holding the styling so it's needed.